astro/src/content/docs/lifecycle/authenticate-users/identity-providers/index.mdx

```diff
@@ -113,6 +114,20 @@ An email address or domain may be provided in the `login_hint` request parameter
 
 You can read more about the `login_hint` and `idp_hint` parameters in the [OAuth Endpoints documentation](/docs/lifecycle/authenticate-users/oauth/endpoints).
 
+## Managed Domains
+
+For SAML v2 or OIDC identity providers, you can optionally enable managed domains. This feature is not available for any other types of identity providers.
+
+If users share a common email domain, such as `@example.com`, you can use managed domains to streamline their login process. Here's how it works:
+
+- On the login page, users are prompted to enter their email address.
+- If the email domain matches a managed domain defined in the Identity Provider settings, the user is automatically redirected to the configured identity provider for authentication.
```

Comment:
Do we have guidelines around when to use “identity provider” vs “Identity Provider” vs “IdP”?

we may not use IdP in documentation, but the new section uses both with and without capitalization. Should they be consistent or are they different uses?

Reply:
Great question. Here's what we have in DevDocsReadme:

> Capitalize all domain objects, especially when working the object's API in which it is created and updated in FusionAuth. For example, see the API Key APIs description for apiKeyId, where API Key is capitalized: The unique Id of the API Key to create. If not specified a secure random UUID will be generated.

> If referring to something that exists as a domain object in FusionAuth, but you are not explicitly referring to an object being created/updated in FusionAuth, use lowercase. Here are some examples: To allow users to log into and use your application, you’ll need to create an Application in FusionAuth.

IdP in general should not be used except in the specific case of SAML, where IdP means something according to the spec.